### PR TITLE
{data}[GCCcore/8.2.0] dill v0.3.0 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by branfosj**

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0c99b1deedc1dc6cdb8aedab0cd2b365 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e01fc54386f4ad3a2557f3b20bba4359 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u05a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/10a94524dbb988e28946e23696f049f5 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/baddca0a667b6acdb073e453a6f13e3b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0828ab3292fac359f0b74c93b9535b8b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u05a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/3cbbf608710342b5a560677163106358 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/b36a591bfd16e8858c44778cb3d17d84 for a full test report.

File: easybuild/easyconfigs/d/dill/dill-0.3.0-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['993409439ebf7f7902d9de93eaa2a395e0446ff773d29f13dc46646482f76906']
+
+builddependencies = [('binutils', '2.31.1')]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = []
```
Comment (Micket) [inline easybuild/easyconfigs/d/dill/dill-0.3.0-GCCcore-8.2.0.eb]: You can remove this line completely

Comment (branfosj) [reply easybuild/easyconfigs/d/dill/dill-0.3.0-GCCcore-8.2.0.eb]: done

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/a877479a15479edfaa0ad8c9dfc359e6 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @branfosj!

**Pull request merged**